samples/ReverseProxy.Direct.Sample/Startup.cs

```diff
@@ -39,16 +41,23 @@ public void Configure(IApplicationBuilder app, IHttpProxy httpProxy)
                 UseCookies = false
             });
 
+            // Setup our own Header transform class
             var transformer = new CustomTransformer(); // or HttpTransformer.Default;
             var requestOptions = new RequestProxyOptions { Timeout = TimeSpan.FromSeconds(100) };
 
             app.UseRouting();
             app.UseEndpoints(endpoints =>
             {
+                // Normally for YARP we would call endpoints.MapReverseProxy() which would use YARP's pipeline to figure out a destination based
+                // on configuration and the stages in the pipeline.
+                // If you have alternatives to those steps, you can skip them and just tell the proxy which destination endpoint to use. It uses the 
+                // HttpClient to forward the request and response, calling into the supplied header transformer. 
```

Comment:
It's hard to say which usage model of YARP is more 'Normal', I think we currently have more customers using IHttpProxy than anything else. This comparison also assumes you're familiar with the other model, where we don't make the opposite comparison in the other samples.
```suggestion
                // When using IHttpProxy for direct proxying you are responsible for routing, destination discovery, load balancing, affinity, etc..
                // For an alternate example that includes those features see BasicYarpSample.
```

Reply:
Our internal guys are probably not normal and the main usage.

Reply:
Fair, but a lot of external customers are using IHttpProxy as well.